database/insertData/03_template_A_featureShowcase.js

```diff
@@ -512,6 +512,7 @@ exports.queries = [
                           operator: "graphQL",
                           children: [
                             "query getOrgs {organisations {nodes {name}}}",
+                            "graphQLEndpoint",
```

Comment:
should this be the end point ?

Reply:
Nah, it's the default, so it'll just use the existing endpoint from the passed-in connection object. Could also be "" or `null`. (Personally I think `""` is better to use in this context -- precisely to avoid the exact confusion you just had  😄)